F/Faiss/bundled/patches/mingw32.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/faiss/impl/platform_macros.h b/faiss/impl/platform_macros.h
+index 2aecc512..9c2e416c 100644
+--- a/faiss/impl/platform_macros.h
++++ b/faiss/impl/platform_macros.h
+@@ -11,7 +11,7 @@
+ #include <cstdint>
+ #include <cstdio>
+ 
+-#ifdef _MSC_VER
++#ifdef _MSC_VER || defined(__MINGW32__)
```

Comment:
This should simply check `_WIN32`: https://learn.microsoft.com/en-us/cpp/preprocessor/predefined-macros?view=msvc-170